packager/app/test/testdata/audio-video-with-codec-switching-encryption-trick-play/output.mpd

```diff
@@ -2,28 +2,14 @@
 <!--Generated with https://github.com/google/shaka-packager version <tag>-<hash>-<test>-->
```

Comment:
Can you revert this file?

I know it was changed by running the script. Unfortunately, it is not deterministic right now..

Reply:
Reverted